app/models/question.rb

```diff
@@ -36,4 +38,8 @@ def total_rates
   def like_ratio
     total_rates == 0 ? 0.to_f : positive_rates.to_f / total_rates
   end
+
+  def convert_correct_answers
```

Comment:
@ManyBaby: This is not really a rating average only a % of feedbacks that are positive